dags/common/utils.py

```diff
@@ -149,3 +150,16 @@ def preserve_cdata(article: str):
 
 def parse_to_ET_element(article: str):
     return ET.fromstring(preserve_cdata(article))
+
+
+def parse_without_names_spaces(xml: str):
```

Comment:
you need it for tests only? if yes, maybe you can move it to the test/utils or something?

Reply:
no, I will use it also in the file processing  DAG